src/moscot/problems/cross_modality/_mixins.py

```diff
@@ -58,29 +58,29 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
     def translate(
             self: CrossModalityTranslationMixinProtocol[K, B],
-            forward:bool = True,
-            **kwargs:Any,
+            forward: bool = True,
+            **kwargs: Any,
     ) -> ArrayLike:
         """
         Translate source or target object.
 
         Parameters
         ----------
-        %(forward)s
-
-        Returns
-        -------
-        %(translation)s
+        forward
+            If `True` computes translation from source object to target object, otherwise backward.
+        kwargs
+            keyword arguments for policy-specific `_apply` method of :class:`moscot.problems.CompoundProblem`.
         
         """
         if forward:
-            self._translation = self[('src', 'tgt')].solution.pull(self.adata_tgt.obsm[self._tgt_attr], scale_by_marginals=True)
+            self._translation = self[('src', 'tgt')].solution.pull(self.adata_tgt.obsm[self._tgt_attr], scale_by_marginals=True, **kwargs)
```

Comment:
if we have an `ExternalStarPolicy` we do not always have `src` and `tgt` as keys of the problem.

Consider doing:
```
def translate(source: K, target: K, forward: bool = True, **kwargs)
```